apps/extension/src/Setup/Common/Completion.tsx

```diff
@@ -20,7 +20,7 @@ type Props = {
   status?: CompletionStatus;
   statusInfo: string;
   parentAccountStore?: AccountStore;
-  shieldedAccount?: DerivedAccount;
+  paymentAddress?: string;
```

Comment:
was just simplifying here - we do not need to track the whole shielded account, only the payment address to display in confirmation